### ISSUE TITLE
Problem with fullscreen video on Mac OS

### ISSUE THREAD

**Issue by torgnywalin** [tauri-apps/wry#423]
**Describe the bug**
When using HTML Video in my app I am not able to get it to play fullscreen on Mac OS.
When checking the HTMLElement it says `webkitSupportsFullscreen: false`

My video tag:
```html
<video src="myurltovideo.mp4" muted="false" autoplay="" controls="" preload="auto" class="player"></video>
```

**Steps To Reproduce**
Insert a video tag, and see if fullscreen is available on Mac OS.

**Expected behavior**
I would expect fullscreen to be available, and also "Picture in picture".

**Screenshots**

**Platform and Versions (please complete the following information):**
OS: MacOS 11.6
Rustc: 1.54.0 (a178d0322 2021-07-26)

**Would you assign yourself to resolve this bug?**
- [ ] Yes
- [x] No

**Additional context**
The app is built with Ionic Angular 5.
Im new to Tauri so if there are configuration needed i might be missing it.
This is my configuration for macOS:

`tauri.bundle.macOS`:

```json
{
        "frameworks": [],
        "minimumSystemVersion": "",
        "useBootstrapper": false,
        "exceptionDomain": "*",
        "signingIdentity": "MyCertificate",
        "entitlements": null
},
```

**Comment by wusyong** [comment]
I found there's ways to set it but it's private API.
https://github.com/WebKit/webkit/blob/main/Source/WebKit/UIProcess/API/Cocoa/WKPreferences.mm#L232-L240

That means the app will be rejected by macOS store if using it.

**Comment by wusyong** [comment]
Fixed by #431
